gem/lib/earlgrey/configure_earlgrey.rb

```diff
@@ -353,7 +354,21 @@ def add_earlgrey_framework(target, framework_ref)
     # rubocop:disable Style/PredicateName
     def has_swift?(target)
       target.source_build_phase.files_references.any? do |ref|
-        SWIFT_FILETYPE == (ref.last_known_file_type || ref.explicit_file_type)
+        file_type = ref.last_known_file_type
+        if file_type.to_s.empty?
+          file_type = ref.explicit_file_type
+        end
+
+        if SWIFT_FILETYPE == file_type
+          return true
+        elsif OBJC_FILETYPE == file_type
```

Comment:
I just re-tested this out. This method does indeed seem to be working correctly.

The issue was the version number, even though it was set to zero, was being returned as true, which led to the Swift bundling being done regardless. I'll change this back!